packer/scripts/install-certstrap.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -o errexit -o xtrace
+
+# Certstrap is used for creating the k8s signing keys, which are necessary to avoid communication bugs on host restarts
+echo "Installing certstrap ..."
+# We run chown in docker to avoid requiring sudo
+systemctl start docker.service
```

Comment:
Need to clean up correctly (stop docker and delete the images &c) to avoid dropping turds in the built image.